policyengine_uk/parameters/gov/hmrc/tax_free_childcare/income_thresholds.yaml

```diff
@@ -0,0 +1,26 @@
+description: The HMRC department requires minimum earnings thresholds for tax-free childcare eligibility based on the age of parents or guardians.
+metadata:
+  period: year
+  type: single_amount 
+  amount_unit: currency-GBP
+  threshold_unit: year
+  label: Tax-free childcare minimum earnings threshold
+  reference:
+    - title: Childcare Payments Act 2014 - c. 28 - section 9 (national minimum wage for 8 hours a week)
```

Comment:
Does the law define this in terms of minimum wage rather than income? If so we should represent it that way.

Reply:
The policy parameter in this case is 8. We then multiply that by the minimum wage, rather than hard-coding the income which is not in the law.